plugins/CoreAdminHome/Commands/InvalidateReportData.php

```diff
@@ -223,4 +239,40 @@ private function getSegmentsToInvalidateFor(InputInterface $input, $idSites)
         }
         return $result;
     }
+
+    private function getSegmentsIdToInvalidateFor(InputInterface $input, $idSites)
+    {
+        $idSegments = $input->getOption('idsegments');
+        $idSegments = trim($idSegments);
+        $idSegments = preg_replace('/\s/', '', $idSegments);
```

Comment:
fyi this preg_replace might not be needed since you already trim all the IDs below?